README.md

```diff
@@ -69,6 +69,10 @@ Your `config.json` file is only required for a few options. Anything you do not
     // Cookie session secret key, make sure to randomize and NOT use default
     // either keyboard mash or a generator https://browserling.com/tools/random-hex
     "sessionSecret": "98ki^e72~!@#(85o3kXLI*#c9wu5l!Z",
+    //when set to true, unlimited sessions are allowed
+    "allowMultipleSessions": true,
+    //if the above is set to false, you can specify how many sessions you would like to allow
+    "maxSessions": 1,
```

Comment:
Minor suggestion: how about setting `maxSessions` to null instead to denote unlimited? Or change `allowMultipleSessions` to `allowUnlimitedSessions`.

Reply:
Yeah it's admittedly a little unclear, but changing it may cause some issues for those that have already added `maxAllowedSessions` to their config.

Reply:
> Minor suggestion: how about setting `maxSessions` to null instead to denote unlimited? Or change `allowMultipleSessions` to `allowUnlimitedSessions`.

I don't think unlimited is a reasonable option tbh, setting to a default of 5 sounds better imo. No reason someone should be logged into more than 5 different devices... at the same time. I'm sure most owners wouldn't want them to be either.